script.js

```diff
@@ -295,3 +302,9 @@ emailval.addEventListener('click', () => {
 err.addEventListener('click', () => {
 	err.classList.add('hidden');
 });
+
+const userDetails = JSON.parse(localStorage.getItem('User'));
+if (userDetails) {
+	emailval.value = userDetails.Email;
+	nameholder.value = userDetails.Name;
+}
```

Comment:
**Changes Required**
- [ ] After filling the form, and not submiting it, the entered values should be saved in the browser. Let us try and add this requirement.

Reply:
Done. I removed the on submit save function. Now data will be saved on the basis of data entered without submitting.